code/modules/holodeck/holodeck_map_templates.dm

```diff
@@ -0,0 +1,204 @@
+	mappath = "_maps/templates/holodeck_spacecheckers.dmm"
+	access_flags = STATION_HOLODECK
+
+/datum/map_template/holodeck/kobayashi
+	name = "Holodeck - Kobayashi Maru"
+	template_id = "holodeck_kobayashi"
+	description = "benis"
+	mappath = "_maps/templates/holodeck_kobayashi.dmm"
+	access_flags = STATION_HOLODECK
+
+/datum/map_template/holodeck/winterwonderland
+	name = "Holodeck - Winter Wonderland"
+	template_id = "holodeck_winterwonderland"
+	description = "benis"
+	mappath = "_maps/templates/holodeck_winterwonderland.dmm"
+	access_flags = STATION_HOLODECK
+
+/datum/map_template/holodeck/photobooth
+	name = "Holodeck - Photobooth"
+	template_id = "holodeck_photobooth"
+	description = "benis"
+	mappath = "_maps/templates/holodeck_photobooth.dmm"
+	access_flags = STATION_HOLODECK
+
+/datum/map_template/holodeck/skatepark
+	name = "Holodeck - Skatepark"
+	template_id = "holodeck_skatepark"
+	description = "benis"
+	mappath = "_maps/templates/holodeck_skatepark.dmm"
+	access_flags = STATION_HOLODECK
+
+//bad evil no good programs
+
+/datum/map_template/holodeck/medicalsim
+	name = "Holodeck - Emergency Medical"
+	template_id = "holodeck_medicalsim"
+	description = "benis"
+	mappath = "_maps/templates/holodeck_medicalsim.dmm"
+	restricted = TRUE
+	access_flags = STATION_HOLODECK
+
+/datum/map_template/holodeck/thunderdome1218
+	name = "Holodeck - 1218 AD"
+	template_id = "holodeck_thunderdome1218"
+	description = "benis"
+	mappath = "_maps/templates/holodeck_thunderdome1218.dmm"
+	restricted = TRUE
+	access_flags = STATION_HOLODECK
+
+/datum/map_template/holodeck/burntest
+	name = "Holodeck - Atmospheric Burn Test"
+	template_id = "holodeck_burntest"
+	description = "benis"
+	mappath = "_maps/templates/holodeck_burntest.dmm"
+	restricted = TRUE
+	access_flags = STATION_HOLODECK
+
+/datum/map_template/holodeck/wildlifesim
+	name = "Holodeck - Wildlife Simulation"
+	template_id = "holodeck_wildlifesim"
+	description = "benis"
+	mappath = "_maps/templates/holodeck_wildlifesim.dmm"
+	restricted = TRUE
+	access_flags = STATION_HOLODECK
+
+/datum/map_template/holodeck/holdoutbunker
+	name = "Holodeck - Holdout Bunker"
+	template_id = "holodeck_holdoutbunker"
+	description = "benis"
+	mappath = "_maps/templates/holodeck_holdoutbunker.dmm"
+	restricted = TRUE
+	access_flags = STATION_HOLODECK
+
+/datum/map_template/holodeck/anthophillia
+	name = "Holodeck - Anthophillia"
+	template_id = "holodeck_anthophillia"
+	description = "benis"
+	mappath = "_maps/templates/holodeck_anthophillia.dmm"
+	restricted = TRUE
+	access_flags = STATION_HOLODECK
+
+/datum/map_template/holodeck/refuelingstation
+	name = "Holodeck - Refueling Station"
+	template_id = "holodeck_refuelingstation"
+	description = "benis"
+	mappath = "_maps/templates/holodeck_refuelingstation.dmm"
+	restricted = TRUE
+	access_flags = STATION_HOLODECK
+
+//flag debug holodeck
+
+/datum/map_template/holodeck/gamer
+	name = "Holodeck - Gamer"
+	template_id = "holodeck_gamer"
+	description = "benis"
```

Comment:
bebnis

Reply:
oh yeah forgot about that, thanks for the heads up